elephant_vending_machine/views.py

```diff
@@ -13,15 +15,18 @@
 
 @APP.route('/run-trial', methods=['POST'])
 def run_trial():
-    """Responds with 'Running {trial_name}' string
+    """
+    Responds with 'Running {trial_name}' string
 
     All requests sent to this route should have a trial_name in
     the query string, otherwise a 400 error will be returned
 
-    Returns:
+    :Returns:
```

Comment:
I think this colon is unnecessary per the google docstring format linked in above comment. I'd try removing it and check that the docs are still generated correctly.